scripts/utils/publishNewVersion.js

```diff
@@ -51,22 +51,18 @@ function publishToNpm(npmTag) {
 }
 
 function releaseDocsVersion(newVersion, previousVersion) {
-  const shouldReplaceWithPreviousVersion = previousVersion && isSameMajorVersion(newVersion, previousVersion);
-  if (shouldReplaceWithPreviousVersion) {
-    removeDocsForVersion(previousVersion);
+  const newDocsVersion = newVersion.split('.', 1).concat('.x');
+  const previousDocsVersion = previousVersion.split('.', 1).concat('.x');
+
+  if (newDocsVersion === previousDocsVersion) {
```

Comment:
Wait, wait, that doesn't work this way in JavaScript. What are you trying to do, @asafkorem ?

Reply:
JK, nice catch. Fixing now.